### PR TITLE
More doc block work

### PR DESCRIPTION
**Pull request by elinw**
Code style

### CODE REVIEW

**Comment by elinw** [comment]
I believe that except for the reference issue this finishes off ParamNameNoMatch.

File: libraries/joomla/html/parameter/element/calendar.php [hunk 1]
```diff
@@ -14,26 +14,32 @@ defined('JPATH_PLATFORM') or die;
...
  * @package     Joomla.Platform
  * @subpackage  Parameter
- * @since       11.1
  * @deprecated  Use JFormFieldCalendar instead.
+ * @since       11.1
  */
 class JElementCalendar extends JElement
 {
 	/**
-	 * @var    string  Element named
```
Comment (LouisLandry) [inline libraries/joomla/html/parameter/element/calendar.php:23]: You completely removed the @var tag here... we still need it.

File: libraries/joomla/html/parameter/element/calendar.php [hunk 2]
```diff
@@ -14,26 +14,32 @@ defined('JPATH_PLATFORM') or die;
...
+	 * @since       11.1
 	 */
 	protected $_name = 'Calendar';
 
 	/**
+	 * Fetch a calendar element
 	 *
+	 * @param   string   $name          Field name
+	 * @param   string   $value         The date value
+	 * @param   object   $node..........JSimpleXMLElement node object containing the settings for the element
```
Comment (LouisLandry) [inline libraries/joomla/html/parameter/element/calendar.php]: I'm fairly certain you didn't mean to have a bunch of dots between $node and JSimpleXMLElement

**Comment by elinw** [comment]
heh

**Comment by elinw** [comment]
There you go.

**Pull request merged**